tutorial/image-focus-point.md

```diff
@@ -0,0 +1,98 @@
+---
+title:  Implementing Focus Point using Gumlet 
+parent: Malibu Tutorial
+nav_order: 27
+nav_exclude: true
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/)_
+
+You can set a focal point to ensure that a key part of an image won't get cropped. Focus points are particularly useful for responsive layouts and make sure the most important part of the image is always visible regardless of the element size. It will ensure that embarrassing image crops do not happen.
```

Comment:
Change this "and make sure the most important part of the image" to "and to  make sure the most important part of the image"

Reply:
done